sw/host/opentitanlib/src/image/manifest.rs

```diff
@@ -154,6 +171,50 @@ pub struct ManifestExtSecVerWrite {
     pub write: u32,
 }
 
+/// Integrator Specific Firmware Binding product expression.
+#[derive(Debug, Serialize, Deserialize, Annotate)]
+pub struct ManifestExtIsfbProductExpr {
+    pub mask: u32,
+    pub value: u32,
+}
+
+/// Integrator Specific Firmware Binding manifest extension.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ManifestExtIsfb {
+    pub header: ManifestExtHeader,
+    pub strike_mask: [u32; 4],
```

Comment:
Do we want to use `u128` here?

Reply:
Done. Thanks, this helped simplify the code.